libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupGoogleFragment.java

```diff
@@ -186,34 +224,49 @@ public void onSocialChanged(OnSocialChanged event) {
             switch (event.error.type) {
                 // WordPress account exists with input email address, and two-factor authentication is required.
                 case TWO_STEP_ENABLED:
+                    AppLog.d(T.MAIN, "GOOGLE SIGNUP: onSocialChanged - error - two step authentication");
                     mAnalyticsListener.trackSignupSocialToLogin();
                     mLoginListener.showSignupToLoginMessage();
                     // Dispatch social login action to retrieve data required for two-factor authentication.
                     PushSocialPayload payload = new PushSocialPayload(mIdToken, SERVICE_TYPE_GOOGLE);
+                    AppLog.d(T.MAIN,
+                            "GOOGLE SIGNUP: onSocialChanged - error - two step authentication - dispatching "
```

Comment:
The first part of the string can be on the previous line with the second part can be on the second since it makes the line longer than 120 characters.